sources/packages/api/src-sql/DisbursementSchedules/Add-tuition-remittance.sql

```diff
@@ -0,0 +1,7 @@
+-- Add tuition_remittance_requested_amount
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
+    COLUMN IF NOT EXISTS tuition_remittance_requested_amount INT NOT NULL DEFAULT 0;
+
+COMMENT ON COLUMN sims.disbursement_schedules.tuition_remittance_requested_amount IS 'Tuition remittance Amount Requested';
```

Comment:
No need to "Amount Requested" to have capital letters and please add the period to the sentence.